.changeset/old-zebras-teach.md

```diff
@@ -0,0 +1,19 @@
+---
+'astro': major
+---
+
+Updates `astro sync` behavior
+
+Astro examples default `tsconfig.json` has been updated to include generated types and exclude your build output. This means that `src/env.d.ts` is only necessary if you have added custom or if you're not using a `tsconfig.json` file.
```

Comment:
```suggestion
Astro's default `tsconfig.json` in starter examples has been updated to include generated types and exclude your build output. This means that `src/env.d.ts` is only necessary if you have added custom configurations or if you're not using a `tsconfig.json` file.
```
We need a noun after "custom..." custom settings? custom configurations? custom types?